incubator/hnc/internal/reconcilers/object_test.go

```diff
@@ -45,6 +45,18 @@ var _ = Describe("Secret", func() {
 		cleanupObjects(ctx)
 	})
 
+	PIt("should propagate objects only to selected namespace", func() {
+		setParent(ctx, barName, fooName)
+		setParent(ctx, bazName, fooName)
+
+		// Create a secret that does NOT propogate to bazName
+		a := map[string]string{"propagate.hnc.x-k8s.io/treeSelect": "!" + bazName}
```

Comment:
May as well make three tests: `select`, `treeSelect` and `none`

Reply:
done